demo/lightning/checkpoint/simulated/multiprocessing_train.py

```diff
@@ -176,53 +238,66 @@ def time_checkpoint_operation(benchmark_strategy: BenchmarkStrategy, distributed
             benchmark_strategy.save_checkpoint(
                 distributed_state_dict, filepath=checkpoint_path)
         elif operation == 'load':
-            benchmark_strategy.load_checkpoint(checkpoint_path=checkpoint_path)
+            benchmark_strategy.load_checkpoint(
+                checkpoint_path=checkpoint_path, initial_state_dict=template_state_dict)
         end_time = time.time()
         times.append(end_time - start_time)
         dist.barrier()  # Synchronize processes
```

Comment:
Do we want the barrier call after the end time? Should it not be prior to end_time? Are we trying to get a timing print out for each node, or a time for the whole operation? If we want timing for each node individually we should specify what node we're running on in the print statement below.

Reply:
Just synced my settings to settings.json

Reply:
Moved the barrier before end_time. Dont need it after since in for loop we already have dist.barrier() before start_time.